packages/fx-core/src/plugins/resource/bot/configs/scaffoldConfig.ts

```diff
@@ -34,10 +43,26 @@ export class ScaffoldConfig {
       rawProgrammingLanguage &&
       utils.existsInEnumValues(rawProgrammingLanguage, ProgrammingLanguage)
     ) {
-      this.programmingLanguage = rawProgrammingLanguage as ProgrammingLanguage;
+      this.programmingLanguage = rawProgrammingLanguage;
     }
 
     this.hostType = ScaffoldConfig.getHostTypeFromProjectSettings(context);
+
+    const rawTriggers = context.answers?.[AzureSolutionQuestionNames.BotNotificationTriggers];
+    if (Array.isArray(rawTriggers)) {
```

Comment:
Would you like to resue the `convertToConstValues`? (May be we can add `typeof value === "string"` into `convertToConstValues`)

Reply:
Updated.

Reply:
(`typeof value === "string"` is redundent in `convertToConstValues` because the right side of the comparison is already a string)